tests/integration_tests.rs

```diff
@@ -226,6 +229,50 @@ fn echo() {
     run_test("echo", 0, input.as_slice(), input.as_slice(), None);
 }
 
+#[test]
+#[serial]
+fn unix_echo() {
+    const FILENAME_IN: &'static str = "/tmp/enarx_unix_echo_test_to_test";
+    const FILENAME_OUT: &'static str = "/tmp/enarx_unix_echo_test_from_test";
```

Comment:
Same comment here as `test_to_test` is more ambiguous.

Reply:
should be resolved